keyboards/idyllic/tinny50_rgb/readme.md

```diff
@@ -0,0 +1,27 @@
+# Tinny50 RGB Edition
+
+![Tinny50 RGB Edition](https://i.imgur.com/TMig65z.jpg)
```

Comment:
```suggestion
![Tinny50 RGB Edition](https://i.imgur.com/TMig65zh.jpg)
```

Reply:
Could I ask the reasoning behind this suggestion?

Reply:
Because this gets Imgur to serve up a smaller, but still high res version of the image, that is better suited for github markdown rendering.

Reply:
Got it, will apply, thanks